src/cuda/cuda-resource-heap.cpp

```
@@ -0,0 +1,54 @@
+    {
+        SLANG_CUDA_ASSERT_ON_FAIL(cuMemFree(m_memory));
+    }
+}
+
+Result ResourceHeapImpl::init()
+{
+    DeviceImpl* device = getDevice<DeviceImpl>();
+    SLANG_CUDA_CTX_SCOPE(device);
+
+    m_isHostMemory = m_desc.memoryType != MemoryType::DeviceLocal;
+    if (m_isHostMemory)
```

Cppcheck (2.21.0)

[style] The function 'create' is never used.

(unusedFunction)